test/generic/pointers/test_pointer_tensor.py

```diff
@@ -379,6 +379,21 @@ def test_get_remote_shape(workers):
     assert y.shape == torch.Size([5])
 
 
+def test_get_remote_ndim(workers):
+    """Test pointer.ndim functionality"""
+    bob = workers["bob"]
+    x = th.rand(2, 3, 4).send(bob)
+    assert x.ndim == 3
+
+
+def test_remote_T(workers):
+    """Test pointer.T functionality"""
+    bob = workers["bob"]
+    x = th.rand(2, 3, 4).send(bob)
+    y = x.T
+    assert y.shape == torch.Size([4, 3, 2])
```

Comment:
Here the same, test "y" not the shape of y

Reply:
doen in a new commit, by testing `bob_xT.get()==x.T` and `bob_x.get()==x`